serverless.yml

```diff
@@ -3,18 +3,58 @@ service: mes-custom-authoriser
 provider:
   name: aws
   runtime: nodejs8.10
+  environment:
+    AZURE_AD_TENANT_ID: 6c448d90-4ca1-4caf-ab59-0a2aa67d7801
+    AZURE_AD_CLIENT_ID: 09fdd68c-4f2f-45c2-be55-dd98104d4f74
+    USERS_DDB_TABLE_NAME: users
+    AWS_REGION: eu-west-1
 
 package:
   individually: true
 
 functions:
   authoriser:
+    handler: local/handler.handler
+    events:
+      - http:
+          path: 'authoriser'
+          method: get
+          authorizer: authoriserFunc 
+  authoriserFunc: 
     handler: src/functions/authoriser/framework/handler.handler
 
 custom:
+  dynamodb:
+    start:
+      port: 8000
+      inMemory: true
+      migrate: true
+      seed: true
+    seed:
+      test:
+        sources:
+          - table: users
+            sources: [./test-data/dynamo-seed-users.json]
   webpack:
     webpackConfig: 'webpack-sls-offline.config.js'
 
+resources:
+  Resources:
+    penaltyDocuments:
```

Comment:
penaltyDocuments is an RSP specific resource name - can we change this to something relevant.

Reply:
Good spot! We need to change the microservice template as well